specification/saas/resource-manager/readme.go.md

```diff
@@ -14,6 +14,6 @@ These settings apply only when `--tag=package-2018-03-01-beta --go` is specified
 Please also specify `--go-sdks-folder=<path to the root directory of your azure-sdk-for-go clone>`.
 
 ```yaml $(tag) == 'package-2018-03-01-beta' && $(go)
-namespace: microsoft.saas
-output-folder: $(go-sdks-folder)/saas/Generated
+namespace: saas
+output-folder: $(go-sdks-folder)/services/preview/$(namespace)/mgmt/2018-03-01-beta/$(namespace)
```

Comment:
```suggestion
output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2018-03-01-preview/$(namespace)
```

Reply:
fixing go-sdks-folder to go-sdk-folder.
We will keep the version as beta for now, as this is mostly an existing API.

We will soon do an overhaul of our RP and will move to new api versions anyway.